test/multiple-element-interactions-test.js

```diff
@@ -42,53 +44,55 @@ describe("When multiple DOM elements are present", () => {
             });
         });
 
-        it("can read attributes from custom child element's prototypes", () => {
-            var DataSource = components.newElement();
-            DataSource.data = [1, 2, 3];
-            components.registerElement("data-source", { prototype: DataSource });
+        // Pending until we decide on a good solution
```

Comment:
Ah, I see. I've checked this out and had a look, and it's just failing because of some bugs in the conversion to the V1 API (there should be two separate element definitions, and they've become only one somehow).

You can see an working version that's equivalent to the original but using CE v1 running in a browser at http://jsbin.com/posemil/edit?html,js,output. If you convert the test to match, it should pass happily.